### PR TITLE
Submitting solution and tests for branch nc-news-6 (Ticket 7)

### PR DESCRIPTION
**Pull request by AnnaTas77**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -196,3 +196,59 @@ describe("GET /api/articles/:article_id/comments", () => {
...
+            body: "The owls are not what they seem.",
+        };
+        return request(app)
+            .post("/api/articles/1/comments")
+            .send(newComment)
+            .expect(201)
+            .then(({ body }) => {
+                const { postedComment } = body;
+                // console.log("postedComment =>>", postedComment);
+
```
Comment (EddTheDuck) [inline __tests__/app.test.js:212]: Please remove console.logs / commented out code

File: __tests__/app.test.js [hunk 2]
```diff
@@ -196,3 +196,59 @@ describe("GET /api/articles/:article_id/comments", () => {
...
+        };
+        return request(app)
+            .post("/api/articles/banana/comments")
+            .send(newComment)
+            .expect(400)
+            .then(({ body }) => {
+                expect(body.msg).toBe("Bad request");
+            });
+    });
+});
```
Comment (EddTheDuck) [inline __tests__/app.test.js:310]: These are good tests but you shoudl also test for the following - 
Status 201, ignores unnecessary properties
Status 404, non existent ID, e.g. 0 or 9999
Status 404, username does not exist

**Review by EddTheDuck — CHANGES_REQUESTED** [review]
Please add the tests mention in my comment and resubmit, thanks

**Review by Dutton1990 — APPROVED** [review]
Super refactor Anna, thanks for adding those new tests. Looks great to me you can merge and continue!

**Pull request merged**